packages/firestore/compat/bundle/package.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I am wondering if we need this since Firestore Compat is meant to be temporary. I wouldn't be opposed to just including Bundle support by default.

And I also think we should include it in the main API in the near future (once all the size-conscious users have migrated).

Reply:
Do you remember how large the bundle code is? I'm a little concerned if people migrate to compat and find the size of their app increases significantly (though it could be a motivation for them to migrate to the modular API).

What do you mean by the main API, in v8? I don't think we are going to touch v8 once v9 is published.

Reply:
Main API - v8.

I think bundles was 30Kb (before gzip).

Reply:
Removed bundle and memory builds. I think it's reasonable to assume size conscious customers will migrate to the modular SDK, and thus providing bundle/memory builds in the compat layer provides limited value while complicates the build.